cairo-auditor/SKILL.md

```
@@ -60,20 +66,111 @@ allowed-tools: [Bash, Read, Glob, Grep, Task]
+   ```bash
+   export SCAN_ID="${SCAN_ID:-cairo-audit-$(date +%Y%m%d)}"
+   export REPO_ROOT="/path/to/repo"
+   export SKILLS_ROOT="${SKILLS_ROOT:-$(git rev-parse --show-toplevel 2>/dev/null || true)}"
+   if [ -z "$SKILLS_ROOT" ] || [ ! -f "$SKILLS_ROOT/scripts/quality/scan_external_repos.py" ]; then
+     echo "ERROR: set SKILLS_ROOT to your starknet-skills checkout root (must contain scripts/quality/scan_external_repos.py)." >&2
+     exit 1
+   fi
+   export BUNDLE_ROOT="/tmp/cairo-auditor/${SCAN_ID}"
+   mkdir -p "$BUNDLE_ROOT"
+
+   python - <<'PY'
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)